packages/typespec-go/.scripts/cadl-ranch.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+import { execSync } from 'child_process';
+
+const toolsModRoot = execSync('git rev-parse --show-toplevel').toString().trim() + '/packages/typespec-go/node_modules/@azure-tools/';
+
+const args = process.argv.slice(2);
+const switches = [];
+for (var i = 0 ; i < args.length; i += 1) {
```

Comment:
Nitpicking: this loop iterates only once and the script silently ignores unexpected extra arguments. Should it just look at `argv[2]` and throw if its value is unexpected or it isn't the only argument?